Pipfile

```diff
@@ -0,0 +1,32 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+flake8 = "*"
+flake8-bandit = "*"
+flake8-bugbear = "*"
+flake8-import-order = "*"
+flake8-string-format = "*"
+flake8-tidy-imports = "*"
+mccabe = "*"
+pep8-naming = "*"
+coverage = "*"
+unittest-xml-reporting = "*"
```

Comment:
this was installed as a CI extra previously because it's not needed on local setups, this needs to be re-added